web/source/osk/visualKeyboard.ts

```diff
@@ -2336,7 +2340,9 @@ namespace com.keyman.osk {
 
         for(Ln=0; Ln<keyboardsList.length; Ln++) {
           if(p == keyboardsList[Ln]['KI'].toLowerCase().replace('keyboard_','')) {
-            PKbd=keyboardsList[Ln]; break;
+            // Requires the Keyboard wrapping object now.
+            PKbd = new com.keyman.keyboards.Keyboard(keyboardsList[Ln]);
+            break;
```

Comment:
Fixes that minor 14.0-only breakage I mentioned.

Reply:
What was breaking? Can you describe the symptoms?

Reply:
Uh, it just wouldn't display due to a `null` reference.  The value now lives as a field of the `Keyboard` wrapper-class now.

Reply:
I obviously forgot to test the documentation-keyboard part when I made the change as part of the Web-core work.